skia-bindings/Cargo.toml

```diff
@@ -48,12 +48,16 @@ textlayout = []
 webp = ["webp-encode", "webp-decode"]
 webp-encode = []
 webp-decode = []
+# sys libraries
+use-system-jpeg-turbo = ["mozjpeg-sys"]
```

Comment:
`mozjpeg` is a fork that is ABI compatible with libjpeg, and can be configured to be the same, but it has a default configuration that leads to superior compression.